### PR TITLE
Delete launch configuration

### PR DESCRIPTION
**Pull request by khanhntd**
**Description:** <Describe what has changed.>

Currently, the integration test account's launch configuration has reached its quota. Therefore, deleting old launch configuration and auto-scaling group is compulsory.

**Link to tracking Issue:** <Issue number if applicable>
N/A

**Testing:** <Describe what testing was performed and which tests were added.>
Testing locally.

**Documentation:** <Describe the documentation added.>
N/A

### CODE REVIEW

File: tools/workflow/clean_aws_resources.go [hunk 1]
```diff
@@ -97,17 +105,172 @@ func terminateEc2Instances() {
...
+
+			//Skipping lc that does not older than 5 days
+			if !time.Now().UTC().Add(pastDayDeleteCalculation).After(*lc.CreatedTime) {
+				continue
+			}
+
+			//Split the instance profile to get the testing ID associate with the testing framework
+			splitLcIamInstanceProfile := strings.Split(*lc.IamInstanceProfile, "-")
+
+			if len(splitLcIamInstanceProfile) != 6 {
```
Comment (jefchien) [inline tools/workflow/clean_aws_resources.go:214]: 6 feels arbitrary. Make it a named const.

**Review by jefchien — COMMENTED** [review]
This clean_aws_resources.go file is getting a bit large. Might be worth splitting it out.

File: tools/workflow/clean_aws_resources.go (hunk 1 above)
Comment (khanhntd) [reply tools/workflow/clean_aws_resources.go]: Reasonable. Changed to ```requiredIAMInstanceProfileLength``` in newest commit. About splitting, agree. Still finding a boilerpoint to do it. Moreover, one problem to address when splitting the file is: should we still used go language and maintaining it dependencies (same as tool linters) or should we use bash? Personally, I would prefer bash but going to far to turn back to use bash now.

Comment (jefchien) [reply tools/workflow/clean_aws_resources.go]: It doesn't hurt to still use go. If we do split it up, it shouldn't be in this PR.

**Review by jefchien — APPROVED** [review]

**Pull request merged**